yarn.lock

```diff
@@ -10533,6 +10533,18 @@ bls12377js@celo-org/bls12377js#566137fd6b9568ef105a3aeaf8e36ff03da91d49:
     ts-node "^8.4.1"
     typescript "^3.6.4"
 
+"bls12377js@https://github.com/celo-org/bls12377js#3755c6b9d27a2d93c82eeb95d7d1dac3ebe321ad":
+  version "0.1.0"
+  resolved "https://github.com/celo-org/bls12377js#3755c6b9d27a2d93c82eeb95d7d1dac3ebe321ad"
+  dependencies:
+    "@stablelib/blake2xs" "0.10.4"
+    "@types/node" "^12.11.7"
+    big-integer "^1.6.44"
+    chai "^4.2.0"
+    mocha "^6.2.2"
+    ts-node "^8.4.1"
+    typescript "^3.6.4"
+
 "bls12377js@https://github.com/celo-org/bls12377js#400bcaeec9e7620b040bfad833268f5289699cac":
```

Comment:
If this version is still here, it implies some package somewhere still relies on that version. Is that an issue?

Reply:
I believe it's old version of utils that depend on it, and some packages depend on old version of utils